web-sender/src/main/java/org/openmrs/eip/web/sender/SenderDashboardGenerator.java

```diff
@@ -58,6 +61,8 @@ public Dashboard generate() {
 		final AtomicInteger totalSyncMsgCount = new AtomicInteger();
 		final Map<String, Map> syncMsgsTableStatsMap = new ConcurrentHashMap();
 		final Map statusItemCountMap = new ConcurrentHashMap();
+		final AtomicInteger totalArchivedCount = new AtomicInteger();
```

Comment:
I think archives should not be on the dashboard because they are less important, most of the time users won't be interested in knowing them

Reply:
Done